### PR TITLE
Replace blacklist wtih default deny and reject

### PR DESCRIPTION
**Pull request by eiais**
switch is more inclusive language

### CODE REVIEW

**Comment by radosroka** [comment]
Hello, Thanks for the contribution. 

Would you squash these commits before I merge them?

**Comment by eiais** [comment]
yep yep, will do this weekend

**Comment by eiais** [comment]
done sorry about the delay

**Comment by radosroka** [comment]
Thanks!

File: doc/man/usbguard-rules.conf.5.adoc [hunk 1]
```diff
@@ -292,7 +292,7 @@ The following set of rules forms a policy that allows USB flash disks and explic
 ....
 +
 The policy rejects all USB flash disk devices with an interface from the HID/Keyboard, Communications and Wireless classes.
-Note that blacklisting is the wrong approach and you shouldn't just blacklist a set of devices and allow the rest.
+Note that default allow is the wrong approach and you shouldn't just reject a set of devices and allow the rest.
```
Comment (genodeftest) [inline doc/man/usbguard-rules.conf.5.adoc:295]: Shouldn't this be `default deny` here?

@radosroka

Comment (genodeftest) [reply doc/man/usbguard-rules.conf.5.adoc:295]: Also @eiais

File: src/Tests/Source/CheckScripts/spell-check.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ if [[ -n "${WORDS}" ]]; then
   echo "========================================"
   echo -e "${WORDS}"
   echo "========================================"
-  echo "To whitelist a word, add it to this dictionary:"
+  echo "To exmpemt a word, add it to this dictionary:"
```
Comment (ZoltanFridrich) [inline src/Tests/Source/CheckScripts/spell-check.sh:55]: what does "exmpemt" mean?

File: doc/man/usbguard-rules.conf.5.adoc (hunk 1 above)
Comment (ZoltanFridrich) [reply doc/man/usbguard-rules.conf.5.adoc:295]: > Shouldn't this be `default deny` here?
> 
> @radosroka

No, it is correctly stated that default allow is the wrong approach

File: src/Tests/Source/CheckScripts/spell-check.sh (hunk 2 above)
Comment (eiais) [reply src/Tests/Source/CheckScripts/spell-check.sh:55]: exempt with a typo. my bad

**Pull request merged**